cmd/lightclient/sentinel/proto/p2p/generate.go

```diff
@@ -0,0 +1,3 @@
+package p2p
+
+//go:generate go run ./p2pgen -i spec_p2p.yaml -o .
```

Comment:
File is useless, please remove

Reply:
where do you want me to move this to? this is where the go generate entry point is, generated and generated_encoding are both generated, so I can't put it there.